scripts/drtprod

```diff
@@ -47,6 +47,9 @@ case $1 in
         set -- start "--binary" "./cockroach" --args="--log=\"file-defaults: {dir: 'logs', max-group-size: 1GiB}\"" --store-count=16 --restart=false "$@"
         roachprod run $cluster -- "\"sudo systemctl unmask cron.service ; sudo systemctl enable cron.service ; echo \"crontab -l ; echo '@reboot sleep 100 && ~/cockroach.sh' | crontab -\" > t.sh ; sh t.sh ; rm t.sh\""
         ;;
+      "drt-ldr1"|"drt-ldr2")
+        set -- start "--binary" "./cockroach" --args="--log=\"file-defaults: {dir: 'logs', max-group-size: 1GiB}\"" --restart=false "$@"
```

Comment:
The quotes may cause some trouble. I had to remove them for `drt-large` https://github.com/cockroachdb/cockroach/pull/126983/files.

Reply:
Good catch! Removed.